core/arch/arm/kernel/thread_spmc.c

```diff
@@ -90,23 +90,27 @@ struct mem_frag_state {
 };
 
 /*
- * If @rxtx_size is 0 RX/TX buffers are not mapped or initialized.
+ * If @ffa_rxtx->size is 0 RX/TX buffers are not mapped or initialized.
  *
- * @rxtx_spinlock protects the variables below from concurrent access
- * this includes the use of content of @rx_buf and @frag_state_head.
+ * @ffa_rxtx->spin_lock protects the variables below from concurrent access
+ * this includes the use of content of @ffa_rxtx->rx and @frag_state_head.
  *
- * @tx_buf_is_mine is true when we may write to tx_buf and false when it is
- * owned by normal world.
+ * @ffa_rxtx->tx_buf_is_mine is true when we may write to ffa_rxtx->tx and
+ * false when it is owned by normal world.
  *
  * Note that we can't prevent normal world from updating the content of
  * these buffers so we must always be careful when reading. while we hold
  * the lock.
  */
-static void *rx_buf;
-static void *tx_buf;
-static unsigned int rxtx_size;
-static unsigned int rxtx_spinlock;
-static bool tx_buf_is_mine;
+
+static struct ffa_rxtx nw_buf = {
```

Comment:
I think `nw_rxtx` instead of `nw_buf` would be more suitable.

Reply:
Done